WordPress/build.gradle

```diff
@@ -29,12 +29,6 @@ repositories {
             includeGroup "com.automattic.tracks"
         }
     }
-    maven {
-        url "https://a8c-libs.s3.amazonaws.com/android/hermes-mirror"
-        content {
-            includeGroup "org.wordpress-mobile"
-        }
-    }
```

Comment:
> It removes the reference to hermes-mirror since it's no longe

I understand that now on we won't need to publish the hermes artifacts as we were doing [here](https://github.com/wordpress-mobile/react-native-mirror-publisher/blob/50d840389524203b22ed2b51e2b28c67f4b9b942/build.gradle.kts#L23-L51) in the `react-native-mirror-publisher` repository, is this accurate? If so, we might update that repository as a follow-up task once this gets merged.

Reply:
>I understand that now on we won't need to publish the hermes artifacts as we were doing [here](https://github.com/wordpress-mobile/react-native-mirror-publisher/blob/50d840389524203b22ed2b51e2b28c67f4b9b942/build.gradle.kts#L23-L51) in the react-native-mirror-publisher repository, is this accurate?

No, it is not. We are publishing the Hermes artifacts but under the `react-native` [repository](https://github.com/wordpress-mobile/react-native-mirror-publisher/commit/006b8c05886f0089c2071b388beff29b2f6483c3) and not the `hermes-mirror` one we used before.

Reply:
Ah great, thanks for the clarification 🙇.